api/authorization-api-1_0.md

```diff
@@ -260,22 +365,23 @@ A Reason Field is a JSON object that has keys and values of type `string`. The f
   "en": "location restriction violation"
 }
 ~~~
+{: #reason-example title="Example Reason"}
 
 ###### Reason Object {#reason-object}
 A Reason Object specifies a particular reason. It is a JSON object that has the following fields:
 
-id:
-: REQUIRED. A numeric value of that specifies the reason within the scope of a particular response
+`id`:
+: REQUIRED. A numeric value of that specifies the reason within the scope of a particular response.
 
-reason_admin:
+`reason_admin`:
```

Comment:
Rather than having different fields for different audiences, why not have a reason field and then an audience field?

Reply:
leaving this in the spec as-is for now, since it doesn't appear to be blocking the merge of the PR and getting us into a better state. we can discuss the reason fields as a separate PR.